VMS/Pages/Admin/AccountManagement/AccountAdminSpManagement.razor

```diff
@@ -0,0 +1,172 @@
+                        <td class="acc__content acc__content--name">@item.Name</td>
+                        <div class="acc__menu dropdown">
+                            <i class="material-icons-outlined icon-menu dropdown-toggle" id="dropdownMenu-@item.Id" data-bs-toggle="dropdown" aria-expanded="false">
+                                more_vert
+                            </i>
+                            <ul class="dropdown-menu" aria-labelledby="dropdownMenu-@item.Id">
+                                <li @onclick="ShowEditAccountAdminSp" class="mb-2 dropdown-menu-edit">Sửa tài khoản</li>
+                                <li @onclick="ShowDeleteAccount" class="dropdown-menu-delete">Xóa tài khoản</li>
+                            </ul>
+                        </div>
+                    </tr>
+
+
+                }
+            }
+
+
+        </table>
+    </div>
+    <div class="d-flex justify-content-center">
+        <Pager CurrentPage="@pagedResult.PageIndex"
+               PageCount="@pagedResult.TotalPages"
+               OnPageChanged="@(async (e)=> { page = e; await HandlePageChanged(); })"
+               ShowPageNumbers="true" />
+    </div>
+</div>
+@code {
+    [CascadingParameter] public IModalService Modal { get; set; }
+
+    private PaginatedList<FakeData> pagedResult;
+    private int page = 1;
+
+    protected override void OnInitialized()
+    {
+        pagedResult = GetFakeData();
+    }
+
+    private async Task HandlePageChanged()
+    {
+        pagedResult = GetFakeData();
+        StateHasChanged();
+        await JS.InvokeVoidAsync("window.scrollTo", 0, 0);
+    }
+
+    private PaginatedList<FakeData> GetFakeData()
+    {
+        int i = 0;
+        var data = Enumerable.Range(0, 20).Select(x => new FakeData
+        {
+            Id = ++i + (page - 1) * 20,
+            CreateDate = new DateTime(2021, 11, 02).ToString("dd/MM/yyyy"),
+            Name = "Tên sinh viên rất chi là dài tin thi thoi, ten dai dai dài dài dài daiiiiiiiiiiiiiiiiiiiiiiiiiiii  daiiiiiiiiiiiiiiiiiiiiii   daiiiiiiiiiiiiiiiiiiiiiiiiiiii ",
+
+        }).ToList();
+
+        var result = new PaginatedList<FakeData>(data, 100, page, 20);
+        return result;
+    }
+
+    private class FakeData
+    {
+        public string Name { get; set; }
+        public int Id { get; set; }
+        public bool IsMenu { get; set; }
+        public string CreateDate { get; set; }
+        public bool IsCheck { get; set; } = false;
+    }
+
+    // Chose all
+    bool IsCheckAll = false;
+    private void ChoseAll()
+    {
+        IsCheckAll = !IsCheckAll;
+        @foreach (var item in pagedResult.Items)
+        {
+            item.IsCheck = IsCheckAll;
+        }
+    }
+    private void DeselectAll()
+    {
+        IsCheckAll = false;
+    }
+
+    // Modal
+
+    private async Task ShowEditAccountAdminSp()
+    {
+        var options = new ModalOptions()
```

Comment:
No need to init modal options anymore. We already have a method to retrieve it, please see `BlazoredModalOptions.GetModalOptions()`.
Example: https://github.com/UEH-Squad/VMS/blob/a261a3ee729d56186d63a53e3249da8fb7afce6d/VMS/Pages/ActivityInfoPage/Rightrail.razor.cs#L92
Please check other places in this PR.

Reply:
Done